ch3/recursion.js

```diff
@@ -0,0 +1,32 @@
+const assert =  require('assert');
+/**
+ * checks if input value is even
+ * @param {*} n - input value
+ * @returns true if even, false if not
+ */
```

Comment:
i like docstrings, but i wouldn't worry about them in the context of an interview since they just take up time! but yes, definitely +1 to docstrings when working.